packages/core/package.json

```diff
@@ -44,6 +44,11 @@
       "require": "./dist/index.js",
       "default": "./dist/index.js"
     },
+    "./error": {
+      "import": "./dist/error/index.mjs",
+      "require": "./dist/error/index.js",
+      "default": "./dist/error/index.js"
+    },
```

Comment:
Is this entrypoint needed? Can't we just let treeshaking do its thing?

Reply:
Sort of just followed what you had with observable and rpc and they had their own entrypoint